components/kyma-environment-broker/internal/model.go

```diff
@@ -418,3 +445,16 @@ func (l ComponentConfigurationInputList) DeepCopy() []*gqlschema.ComponentConfig
 	}
 	return copiedList
 }
+
+func serviceManagerRequestCereds(parameters ProvisioningParameters) *servicemanager.Credentials {
```

Comment:
I suppose it should be
```suggestion
func serviceManagerRequestCreds(parameters ProvisioningParameters) *servicemanager.Credentials {
```